sources/Services/Midi/MidiService.cpp

```diff
@@ -146,9 +144,14 @@ void MidiService::stopDevice() {
 void MidiService::OnPlayerStart() {
 
   if (deviceName_.size() != 0) {
+    // printf("ReStart Midi DEVICE NAME:%s", deviceName_.c_str());
     stopDevice();
     startDevice();
-    deviceName_ = "";
+    deviceName_ = "MIDI OUT 1";
```

Comment:
shouldn't device name be set up before start? I think original code clears it after start to leave it blank for next run, but may be mistaken.